src/plugins/yamlcpp/yamlcpp/Tree.graffle/data.plist

```diff
@@ -0,0 +1,1348 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file used on the target? (Test files are installed) Maybe put it to doc?

Reply:
Commit 044a71d9 fixes this issue.